Content.Server.Database/Migrations/Postgres/20230222184825_AdminNotesImprovement.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Content.Server.Database.Migrations.Postgres
+{
+    public partial class AdminNotesImprovement : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropIndex(
+                name: "IX_trait_profile_id",
+                table: "trait");
+
+            migrationBuilder.RenameColumn(
+                name: "shown_to_player",
+                table: "admin_notes",
+                newName: "secret");
```

Comment:
Aren't these inverted purposes? Looks like it will require a migration?

Reply:
They are, but they were not used anywhere, so I didn't migrate it

Reply:
I guess if no forks have it?